src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.html

```diff
@@ -1,6 +1,11 @@
-<div
-  class="label"
-  [style.margin-left.px]="labelOffsetInPixels"
->
-  <span #labelTextElement>{{piece.name}}</span>
+<div class="label" [style.margin-left.px]="labelOffsetInPixels">
+  <span #labelTextElement>{{ piece.name }}</span>
+  <sofie-piece-hover-scrub
+    *ngIf="shouldShowHoverScrub()"
+    [pieceLifespan]="piece.lifespan"
+    [fileName]="piece.name"
+    [type]="getTv2PieceMetadata.type"
```

Comment:
This component is called "sofie-_piece_-hover-scrub", but it doesn't take a Piece as an input...
Instead it takes several attributes from the Piece indenpendently. 
It should just take a Piece as an input since the component name says it's a hoverscrub for a _piece_. Lets utilize that we have objects to work with.

Also, the same rules applies for input attributes as applies for parameters. The fewer, the better.

Reply:
Disappears after structure logic changes